### PR TITLE
Populate some empty predicates

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/equipment-effects/effect-greater-potion-of-stable-form.json [hunk 1]
```diff
@@ -23,15 +23,17 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "stableFormTransmutation",
-                "predicate": [],
+                "label": "{item|name} Transmutation",
```
Comment (simonward) [inline packs/equipment-effects/effect-greater-potion-of-stable-form.json:26]: Not sure it's worth localizing these?

File: packs/extinction-curse-bestiary/deghuun-child-of-mhar.json [hunk 2]
```diff
@@ -518,7 +518,9 @@
                 "rules": [
                     {
                         "key": "Note",
-                        "predicate": [],
+                        "predicate": [
```
Comment (simonward) [inline packs/extinction-curse-bestiary/deghuun-child-of-mhar.json:521]: Maybe the predicates for Notes should be removed?

File: packs/equipment-effects/effect-potion-of-stable-form.json [hunk 3]
```diff
@@ -23,15 +23,17 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "stableFormTransmutation",
-                "predicate": [],
+                "label": "Potion of Stable Form (Transmutation)",
```
Comment (simonward) [inline packs/equipment-effects/effect-potion-of-stable-form.json:26]: Not sure it's worth localizing these?

Comment (TikaelSol) [reply packs/equipment-effects/effect-potion-of-stable-form.json:26]: Not for consumables right now, no.

**Pull request merged**